ragna/deploy/_ui/left_sidebar.py

```diff
@@ -79,98 +62,30 @@ def __panel__(self):
         self.chat_buttons = []
         for chat in self.chats:
             button = pn.widgets.Button(
-                name=chat["metadata"]["name"], button_style="outline"
+                name=chat["metadata"]["name"],
+                button_style="outline",
```

Comment:
Shouldn't that also be part of the CSS rather than mixing panel functionality with our custom CSS?